### DISCUSSION TITLE
Remote Images

### DISCUSSION THREAD

**Discussion by iammattmartin** [outline/outline#7093] [General] [answered]
For specific docs that we need to store, we want to use external (to Outline) images. Whilst there are a lot of object types, there isn't anything as simple as just what would be a normal <img> tag.

The image itself come from another system we have but is dynamic which is why it needs to be linked (to view) instead of to click.

Is it possible to support this additional media type in the future, if not already possible?

**Comment by tommoor** [comment, marked as answer]
Hi @iammattmartin – there is a workaround for this limitation, you can format the image in Markdown and then paste that into Outline to get the remote reference, like:

`![image](https://myremoteimagepathpng)`

**Comment by iammattmartin** [comment]
As perfect, that'll do for now - thanks!
